engine/framework/worker.go

```diff
@@ -94,6 +94,7 @@ type BaseWorker interface {
 	// When `err` is not nil, the status code is assigned WorkerStatusError.
 	// Otherwise worker should set its status code to a meaningful value.
 	Exit(ctx context.Context, status frameModel.WorkerStatus, err error) error
```

Comment:
Why we need two Exit interface method?

Reply:
As a user, should I really care about this stuff?
I think the framwork has some promise about the correct time sequence of callback, so the user only need to implement the callback.

Reply:
> As a user, should I really care about this stuff?

No, you shouldn't. This is not a call back, and the user does not need to implement it. The framework provides an implementation.

Reply:
Oh, it's my fault for not making me clear.
I mean: 
Should user use the `NotifyExit`? Or this interface method will be hidden after we refine the BaseMaster/MasterImpl interface?

Reply:
The method is hidden. `Runner` is not exposed to the user.